### PR TITLE
Adds post-apoc prices to all melee weapons (part 4 of reworking all prices)

### PR DESCRIPTION
**Pull request by Brian-Otten**
#### Summary
```SUMMARY: Balance "adds and balances price values on all melee weapons"```

#### Purpose of change
The post apocalypse economy and pricing was completely out of whack and needed a full pass, i'm starting work on pricing.
It's going to be hard to compare prices for now because the current prices are so all over the place, so i'm going to have to ask for a little faith that once more of the PR's add up things will click into place and make sense.

#### Describe the solution
adds price_postapoc to every melee weapon (at least those listed in the melee weapons folder.).

### CODE REVIEW

File: data/json/items/melee/swords_and_blades.json [hunk 1]
```diff
@@ -375,6 +394,7 @@
     "name": { "str": "jian", "str_pl": "jian" },
     "description": "This is a dull, cheaply made replica of an ancient Chinese doubled-edged straight sword, with an ornate guard.",
     "price": 10000,
+    "price_postapoc": 500,
     "price_postapoc": 0,
```
Comment (kevingranade) [inline data/json/items/melee/swords_and_blades.json:398]:
```suggestion
```

File: data/json/items/melee/swords_and_blades.json [hunk 2]
```diff
@@ -432,6 +453,7 @@
     "name": { "str": "scimitar" },
     "description": "This is a dull, cheaply made replica of a curved sword, associated with various Middle Eastern and Central Asian countries.",
     "price": 9300,
+    "price_postapoc": 50,
     "price_postapoc": 0,
```
Comment (kevingranade) [inline data/json/items/melee/swords_and_blades.json:457]:
```suggestion
```

File: data/json/items/melee/swords_and_blades.json [hunk 3]
```diff
@@ -470,6 +493,7 @@
     "name": { "str": "estoc" },
     "description": "This is a thin thrusting sword, a sort of predecessor to the rapier.  It requires a larger baldric or scabbard, compared to smaller swords.  It seems a bit too bendy.",
     "price": 13500,
+    "price_postapoc": 750,
     "price_postapoc": 0,
```
Comment (kevingranade) [inline data/json/items/melee/swords_and_blades.json:497]:
```suggestion
```

File: data/json/items/melee/swords_and_blades.json [hunk 4]
```diff
@@ -489,6 +513,7 @@
     "name": { "str": "estoc" },
     "description": "This is a dull, cheaply-made replica of a thin thrusting sword predating the rapier.  It requires a larger baldric or scabbard, compared to smaller swords.",
     "price": 13500,
+    "price_postapoc": 50,
     "price_postapoc": 0,
```
Comment (kevingranade) [inline data/json/items/melee/swords_and_blades.json:517]:
```suggestion
```

File: data/json/items/melee/swords_and_blades.json [hunk 5]
```diff
@@ -924,6 +966,7 @@
     "name": { "str": "fencing foil" },
     "description": "A weapon used for fencing, the most noble of all sports.  Unfortunately, a fencing foil is rather useless as a weapon, due to its flexible shaft and dull tip.",
     "price": 2000,
+    "price_postapoc": 10,
     "price_postapoc": 0,
```
Comment (kevingranade) [inline data/json/items/melee/swords_and_blades.json:970]:
```suggestion
```

File: data/json/items/melee/swords_and_blades.json [hunk 6]
```diff
@@ -1145,6 +1193,7 @@
     "name": { "str": "cutlass", "str_pl": "cutlasses" },
     "description": "This is a dull, cheaply made replica of a broad saber known for its use by sailors and pirates.",
     "price": 9600,
+    "price_postapoc": 50,
     "price_postapoc": 0,
```
Comment (kevingranade) [inline data/json/items/melee/swords_and_blades.json:1197]:
```suggestion
```

File: data/json/items/melee/swords_and_blades.json [hunk 7]
```diff
@@ -1297,6 +1351,7 @@
     "name": { "str": "broadsword" },
     "description": "This is a dull, cheaply made replica of an early modern sword seeing use in the 16th, 17th, and 18th centuries.  Called 'broad' to contrast with the slimmer rapiers.",
     "price": 12000,
+    "price_postapoc": 50,
     "price_postapoc": 500,
```
Comment (kevingranade) [inline data/json/items/melee/swords_and_blades.json:1355]:
```suggestion
```

**Comment by Brian-Otten** [comment]
Thanks for the review kevin, apparently i forgot to push my duplicate removal PR, but i think you might have caught one i missed. merged our fixes.

**Comment by kevingranade** [comment]
No problemo, it was pretty quick.

**Pull request merged**